scraper.rb

```diff
@@ -40,6 +82,89 @@ class Governor < Scraped::HTML
   field :party do
     noko.css('td')[4].text.tidy
   end
+
+  # Get extra fields from the Pombola data, if present:
+
+  field :email do
+    pombola_contact_details_of_type('email')
+  end
```

Comment:
Sure - I've added a commit which does some stricter checking of the various contact details (and which makes some attempt to make sure URLs have a protocol, etc.)